src/style.css

```diff
@@ -1,3 +1,68 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+header {
+  display: flex;
+  position: relative;
+  z-index: 1;
+  align-items: center;
+  padding: 0 50px;
+  justify-content: space-between;
+  background-color: #fff;
+}
+
+.header__logo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+.nav__item:not(:first-child) {
+  margin-left: 20px;
+}
+
+.nav__link {
+  position: relative;
+  display: flex;
+  align-items: center;
+  height: 60px;
+  font-size: 12px;
+  font-weight: 500;
+  white-space: nowrap;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #000;
+  transition: color 0.3s;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.nav__link.is-active {
+  color: #00acdc;
+}
+
+.nav__list li.nav__item a {
+  position: relative;
+}
+
+.nav__list li.nav__item a.is-active::after {
```

Comment:
don't increase specificity when it's not necessary
1'st point in checklist

Reply:
Thank you! I've already encountered this in my subsequent works, but I didn't notice it here.